include/eth_internals.c

```diff
@@ -277,3 +306,70 @@ bool U4BE_from_parameter(const uint8_t* parameter, uint32_t* value) {
 
     return false;
 }
+
+
+void call_app_ethereum() {
+    unsigned int libcall_params[5];
+    libcall_params[0] = (unsigned int) "Ethereum";
+    libcall_params[1] = 0x100;
+    libcall_params[2] = RUN_APPLICATION;
+    libcall_params[3] = (unsigned int) NULL;
+#ifdef HAVE_NBGL
+    caller_app_t capp;
+    const char name[] = APPNAME;
+    nbgl_icon_details_t icon_details;
+    uint8_t bitmap[sizeof(ICONBITMAP)];
+
+    memcpy(&icon_details, &ICONGLYPH, sizeof(ICONGLYPH));
+    memcpy(&bitmap, &ICONBITMAP, sizeof(bitmap));
+    icon_details.bitmap = (const uint8_t *) bitmap;
+    capp.name = (const char *) name;
+    capp.icon = &icon_details;
+    libcall_params[4] = (unsigned int) &capp;
+#else
+    libcall_params[4] = (unsigned int) NULL;
+#endif
+    os_lib_call((unsigned int *) &libcall_params);
+}
+
+__attribute__((section(".boot"))) int main(int arg0) {
+    // Exit critical section
+    __asm volatile("cpsie i");
+
+    os_boot();
+
+    check_api_level(CX_COMPAT_APILEVEL);
```

Comment:
I think this should be dropped, this is a remaining of the past which has not been update since a long time..

Reply:
ok